inc/auth.php

```diff
@@ -22,6 +22,8 @@
 use phpseclib3\Crypt\Common\SymmetricKey;
 use phpseclib3\Exception\BadDecryptionException;
 
+const UNUSABLE_PASSWORD = '!unusable';
```

Comment:
I would rather not have this global constant here. It could be moved to `inc/defines.php`, or it could be a public constant `dokuwiki\Extension\AuthPlugin`?